expression/aggregation/agg_to_pb.go

```diff
@@ -36,7 +36,11 @@ func AggFuncToPBExpr(sc *stmtctx.StatementContext, client kv.Client, aggFunc *Ag
 	var tp tipb.ExprType
 	switch aggFunc.Name {
 	case ast.AggFuncCount:
-		tp = tipb.ExprType_Count
+		if aggFunc.IsMppFinal {
```

Comment:
I mean pass "isMpp" to this function as an argument, because if any function is in mpp mode, all of them will be.

Reply:
updated.